src/main/java/model/Cars.java

```diff
@@ -0,0 +1,73 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static util.Util.generateRandomNumber;
+
+public class Cars {
+
+    private static final int MIN_CAR_NAME_COUNT = 2;
+    private static final int MIN_POSITION = 0;
+
+    private final List<Car> cars;
+
+    private Cars(final List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public static Cars from(final List<String> names) {
+        validateNames(names);
+
+        List<Car> cars = names.stream()
+                .map(Car::from)
+                .collect(Collectors.toList());
+
+        return new Cars(cars);
+    }
+
+    private static void validateNames(List<String> names) {
+        checkDuplicatedName(names);
+        checkNameCount(names);
+    }
+
+    private static void checkDuplicatedName(List<String> names) {
+        long nameCount = names.stream().distinct().count();
+        if (names.size() != nameCount) {
+            throw new IllegalArgumentException("[ERROR] 자동차의 이름은 중복될 수 없습니다.");
+        }
+    }
+
+    private static void checkNameCount(List<String> names) {
+        if (names.size() < MIN_CAR_NAME_COUNT) {
+            throw new IllegalArgumentException("[ERROR] 자동차 이름은 2개 이상이어야 합니다.");
+        }
+    }
+
+    public void moveCars() {
+        for (Car car : this.cars) {
+            car.moveForward(generateRandomNumber());
+        }
+    }
```

Comment:
```suggestion
    public void moveCars() {
        for (final Car car : this.cars) {
            car.moveForward(generateRandomNumber());
        }
    }
```
레모네가 페어와 final 에 대해 리뷰를 남겨서, for 문에 이런 방식으로 붙일 수도 있다는 것도 알려드리고 싶었어요
반영하시지 않아도 돼요!